src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOnSpecialOasis.java

```diff
@@ -12,6 +12,7 @@
 public class TestAdsOnSpecialOasis extends TemplateNoFirstLoad {
   static final String PROJECT43_TEST_LINE_ITEM_ID = "271491732";
   static final String FILE_PAGE_AD_UNIT = "wka.life/_project43//file";
+  static final String FILE_PAGE_BOTTOM_LB_AD_UNIT = "wka1a.PF/bottom_leaderboard/desktop/oasis-file/_project43-life";
```

Comment:
How about naming it `FILE_PAGE_BLB_MEGA_AD_UNIT` instead? Thanks to such name we'll connect it with the gradual MEGA rollout and one day we'll be able to remove it and come with an idea of verifying MEGA ad units for all slots on a file page.